jmix-flowui/flowui/src/main/java/io/jmix/flowui/impl/DialogsImpl.java

```diff
@@ -769,6 +769,8 @@ public DialogWindow<InputDialog> build() {
     public class BackgroundTaskDialogBuilderImpl<T extends Number, V> implements BackgroundTaskDialogBuilder<T, V> {
 
         protected Dialog dialog;
+        protected VerticalLayout layout;
+        protected Component content;
```

Comment:
It makes no sense to store both `content` and `messageSpan` as a separate fields since we can't display them both as the same time. Take a look at the other dialogs and how `text`/`content` are handled.